### ISSUE TITLE
How can I reproduce the results in the paper?

### ISSUE THREAD

**Issue by butterluo** [multimodal-art-projection/AutoKaggle#4]
How can I reproduce the results in the paper? Just run run_multi_agents.sh?

**Comment by batuhanmtl** [comment]
> How can I reproduce the results in the paper? Just run run_multi_agents.sh?

Yes, after setting up the necessary environment, it is enough to run "run_multi_agents.sh". By default, it will run 5 experiments for each competition.
